### ISSUE TITLE
Bacon texture with white background

### ISSUE THREAD

**Issue by Martin414** [Ellpeck/ActuallyAdditions#1306]
Texture doesn't have transparent back ground and it really bothers me.
![2020-02-25_21 53 34](https://user-images.githubusercontent.com/52965556/75286798-6fb85700-5819-11ea-8634-fd99329dcce7.png)

**Comment by justinbooj9** [comment]
i think you probably tried to edit the texture in paint included with windows. paint does not support transparency so if you open a file with transparent pixels, it converts them into white pixels

**Comment by AlasICannotSwim** [comment]
No, I can confirm that this issue isn't because he opened it in paint. 

I have the same issue, and I've never touched the AA textures. I've had this issue in every modpack I've used that has AA, as well as when I've installed AA manually, yet it seems like only a few people have the bug. 

My first guess was that the texture itself was messed up, but when I looked in the AA jar file, the bacon PNG does actually have a transparent background.

So I've got no idea what the cause could be.
